CHANGELOG.md

```diff
@@ -54,6 +54,15 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 * (kvstore) [\#7415](https://github.com/cosmos/cosmos-sdk/pull/7415) Allow new stores to be registered during on-chain upgrades.
 
+## [v0.40.0-rc0](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.40.0-rc0) - 2020-10-13
```

Comment:
I think this line should be removed, and the new changelog entry added to the "Unreleased" section. We'll take care of the rc1 changelog when actually performing the backport.

Reply:
Oh, I wanted to make it `v0.40.0-rc0`. Should I change to `unreleased`?

Reply:
done, moved to unreleased.